ploop.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
systemd-cat isn't available too. For busybox, we will need another script:
"""
#!/bin/sh

exec 3>&1

`dirname $0`/ploop.bin -logtostderr -- ploop "$@" &> /var/log/ploop-flexvol.log
"""